src/DeploymentProvider/DeploymentHelper.ts

```diff
@@ -232,6 +236,39 @@ export class DeploymentHelper {
             core.debug('Environment Variables: ' + JSON.stringify(transformedEnvironmentVariables));
             deploymentSettingsPart["environmentVariables"] = transformedEnvironmentVariables;
         }
+        const disableProbe: asa.Probe = {
+            disableProbe: true
+        }
+        if (params.enableLivenessProbe.length > 0) {
```

Comment:
If we do not provide `enableLivenessProbe`, I presume it will be `null` and we are likely to get NPE here, right?

Reply:
The `params.enableLivenessProbe` won't be null.
The getInput function is defined as follows:
```
/**
 * Gets the value of an input.
 * Unless trimWhitespace is set to false in InputOptions, the value is also trimmed.
 * Returns an empty string if the value is not defined.
 *
 * @param     name     name of the input to get
 * @param     options  optional. See InputOptions.
 * @returns   string
 */
export declare function getInput(name: string, options?: InputOptions): string;
```
So we will get empty string if no input. Here if condition is to distinguish whether user input an param that we need to override the probe.